x-pack/plugin/voting-only-node/src/main/java/org/elasticsearch/cluster/coordination/VotingOnlyNodeFeatureSet.java

```diff
@@ -43,7 +43,7 @@ public String name() {
 
     @Override
     public boolean available() {
-        return licenseState != null && licenseState.isAllowed(Feature.VOTING_ONLY);
+        return licenseState != null && licenseState.checkFeature(Feature.VOTING_ONLY);
```

Comment:
This method does not seem to be used anywhere? But the inner class `VotingOnlyNodeFeatureSet#UsageInfoAction` also has a `availabe()` method which internally uses `licenseState.checkFeature`. But all other `XPackInfoFeatureTransportAction` uses `licenseState.isAllowed`. Is this inconsistency intended?

Reply:
I've fixed the checkFeature usage, it was incorrect. I'll address the unused class in a followup.